hphp/hack/src/options/globalOptions.mli

```diff
@@ -224,6 +224,12 @@ type t = {
    * New style: xhp class name {}
    *)
   po_enable_xhp_class_modifier: bool;
+  (*
+   * Flag to disable the old stype xhp element mangling. <something/> would otherwise be resolved as xhp_something
+   * The new style xhp class something {}; does not do this style of mangling, thus we need a way to disable it on the
+   * 'lookup side'.
```

Comment:
This is hard to read without code delimiters. How about something like this?

```
   * Flag to disable the old stype xhp element mangling. `<something/>` would otherwise be resolved as `xhp_something`
   * The new style `xhp class something {}` does not do this style of mangling, thus we need a way to disable it on the
```